src/System.Data.SqlClient/ref/System.Data.SqlClient.Manual.cs

```diff
@@ -7,6 +7,25 @@
 
 namespace System.Data.SqlClient
 {
+    // Declaring members from stripped base class CollectionBase
+    public sealed partial class SqlBulkCopyColumnMappingCollection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
Is `Capacity` a commonly used property on `SqlBulkCopyColumnMappingCollection`? None of the other collections in CoreFX that implement `CollectionBase` in the full framework include it.